### PR TITLE
add otel configuration examples

### PR DESCRIPTION
**Pull request by vikramraman**
Adds an example around overriding OTEL collector configuration

### CODE REVIEW

File: examples/traces/values-config.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  # overrides collector configuration
+  config:
+    processors:
+      # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md
+      memory_limiter:
+        # to use percentagles instead of the _mib values
+        limit_percentage: 75
+        spike_limit_percentage: 20
+        # may need to explicitly unset default values specified
```
Comment (bendrucker) [inline examples/traces/values-config.yaml:27]:
```suggestion
        # unset default values
```

Not _may_, _must_:

https://github.com/observeinc/helm-charts/blob/44ad42766b0157a5c13ce1466e1cbcd0af2ef2cb/charts/traces/values.yaml#L145-L147

How the application handles both % and absolute values in config would take further looking but it's at best ambiguous and preferably forbidden.

**Review by bendrucker — APPROVED** [review]

**Comment by jnjackins** [comment]
Would it make sense to put this information in a README file in the otel directory, instead?
I'm thinking that people might not think to go poking around files in `examples` except for when we explicitly tell them to (like for sizing).

File: examples/traces/values-config.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    requests:
+      cpu: 250m
+      memory: 512Mi
+
+  # overrides collector configuration
+  config:
+    processors:
+      # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiterprocessor/README.md
+      memory_limiter:
+        # to use percentagles instead of the _mib values
```
[the remark this answers is not in the document]
Comment (vikramraman) [reply examples/traces/values-config.yaml]: Thanks! Fixed the typo.

**Comment by vikramraman** [comment]
Renamed the file as per offline discussions to `values-memlimit.yaml`. 

I do think a concrete readme in the OTEL directory makes sense. But leaving that as a follow up. For now a good set of examples for common use cases is useful to have.

**Pull request merged**